### ISSUE TITLE
The down arrow on the first page should animate the scroll down

### ISSUE THREAD

**Issue by mushon** [shenkarlab/Off-The-record#21]

**Comment by nir-jackson** [comment]
hey @mushon 
i've been going over the site and i see all the bugs. right now i'n busy but i'll get to work on fixing everything as soon as i'm free (around after noon).

first i'm going to fix critical bugs and then add new features (animation for scroll arrow for example)

**Comment by mushon** [comment]
Yes, issues that are critical appear under the 'Launch' milestone: https://github.com/shenkarlab/Off-The-record/milestones/Launch and others do not. @shenkarlab/off-the-record please go in, test the site (http://viz.calcalist.co.il/otr/web/) and document all the bugs like I have been doing so we can make sure none of them are overlooked before the launch tomorrow(!) Please also close old issues that have already been fixed so we're clear on the tasks we still need to complete.
